pkg/skaffold/build/tag/custom_template.go

```diff
@@ -24,34 +24,34 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-// templateTagger implements Tagger
-type templateTagger struct {
+// CustomTemplateTagger implements Tagger
+type CustomTemplateTagger struct {
```

Comment:
did you intentionally make this public? i don't think it needs to be right?

Reply:
Good catch.